libs/application/templates/inheritance-report/src/fields/ReportFieldsRepeater/index.tsx

```diff
@@ -90,15 +100,28 @@ export const ReportFieldsRepeater: FC<
       return
     }
 
-    const total = values.reduce(
-      (acc: number, current: any) =>
-        Number(acc) + Number(current[props.sumField]),
-      0,
-    )
+    const total = values.reduce((acc: number, current: any) => {
```

Comment:
For these `any`s. Should we include a linter rule to disable warnings just to keep the number of them down?